core/templates/dev/head/profile/profile.html

```diff
@@ -49,13 +49,214 @@ <h1 class="oppia-profile-username"><strong>{{PROFILE_USERNAME}}</strong></h1>
+                            <span class="glyphicon glyphicon-pencil protractor-test-edit-exploration"
+                                  title="Edit (as moderator)"></span>
+                              Edit (as moderator)
+                        </a>
+                      {% endif %}
+
+                      <a href="/explore/<[exploration.id]>" class="oppia-gallery-tile-container-link" style="text-decoration: none;">
+                        <div class="oppia-gallery-tile-contents">
+                          <span class="oppia-gallery-tile-image-container"
+                                style="position: absolute;">
+                            <img class="oppia-gallery-tile-image" ng-src="<[exploration.thumbnail_image_url]>">
+                          </span>
+
+                          <div class="oppia-gallery-tile-details">
+                            <div class="oppia-gallery-tile-first-row">
+                              <span class="oppia-gallery-tile-title protractor-test-gallery-tile-title">
+                                <[exploration.title]>
+                              </span>
+                              <span ng-if="exploration.objective" class="hidden-xs">
+                                <span style="font-weight: bold;">&middot;</span>
+                                <span style="color: #888; font-size: 0.9em;" class="protractor-test-exploration-objective">
+                                  <[exploration.objective | truncateAndCapitalize: 120]>
+                                </span>
+                              </span>
+                            </div>
+
+                            <div class="oppia-gallery-tile-second-row">
+                              <span style="color: #888">
+                                <strong><[exploration.category]></strong>
+                                <span style="font-weight: bold;">&middot;</span>
+                                Last updated <[getLocaleAbbreviatedDatetimeString(exploration.last_updated)]>
+                              </span>
+                              <span ng-if="exploration.status === 'publicized'">
+                                <span style="font-weight: bold;">&middot;</span>
+                                <span style="color: #888">Featured</span>
+                              </span>
+                              <span ng-if="areRatingsShown(exploration.ratings)">
+                                <span style="font-weight: bold;">&middot;</span>
+                                <span class="oppia-gallery-tile-ratings" style="color: black;">
+                                  <rating-from-frequencies rating-frequencies="exploration.ratings"></rating-from-frequencies>
+                                </span>
+                              </span>
+                            </div>
+                          </div>
+                        </div>
+                      </a>
+                    </md-card>
+                  </div>
```

Comment:
Something's gone completely wrong with the indentation here -- the lines below don't match up